IDEAS/Buildings/Components/BaseClasses/ConductiveHeatTransfer/Examples/MultiLayer.mo

```diff
@@ -40,7 +40,19 @@ model MultiLayer "Unit test for multi layer model"
     energyDynamics=Modelica.Fluid.Types.Dynamics.DynamicFreeInitial)
                           "Test model containing only solid layers"
     annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
-
+initial equation
+  // manually adding initial conditions to avoid conflict
+  // with prescribedTemperature blocks
+  cavityWall.monLay[1].monLayDyn.T[1]=cavityWall.T_start[1];
+  cavityWall.monLay[3].monLayDyn.T[1]=cavityWall.T_start[3];
+  cavityWall.monLay[3].monLayDyn.T[2]=cavityWall.T_start[3];
+  cavityWall.monLay[3].monLayDyn.T[3]=cavityWall.T_start[3];
+  cavityWall.monLay[4].monLayDyn.T[1]=cavityWall.T_start[4];
+  cavityWall.monLay[4].monLayDyn.T[2]=cavityWall.T_start[4];
+  TABS.monLay[1].monLayDyn.T[1]=TABS.T_start[1];
+  TABS.monLay[2].monLayDyn.T[1]=TABS.T_start[2];
+  TABS.monLay[3].monLayDyn.T[1]=TABS.T_start[3];
+  TABS.monLay[4].monLayDyn.T[1]=TABS.T_start[4];
```

Comment:
how come this can not be done with the global paramet T_start?

Reply:
Which one? This is the highest level parameter that I could find.